### PR TITLE
Update recovering-your-ssh-key-passphrase.md

### LINKED ISSUE #21366: recovering-your-ssh-key-passphrase.md is outdated
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/authentication/troubleshooting-ssh/recovering-your-ssh-key-passphrase

### What part(s) of the article would you like to see updated?

Changing 

> ... [switch to HTTPS cloning](https://docs.github.com/en/github/getting-started-with-github/managing-remote-repositories) so you can use your GitHub **password** instead.

into

> ...[switch to HTTPS cloning](/github/getting-started-with-github/managing-remote-repositories) so you can use a **Personal Access Token** instead.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ivyturner**
### Why:

Closes #21366 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

changing out the word "password" for "personal access token" as authenticating over HTTPS for Git actions can no longer be done with a password

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@lxjv Thank you for opening a PR and linking it to your issue! ✨ 

I'll get this triaged for review ⚡

File: content/authentication/troubleshooting-ssh/recovering-your-ssh-key-passphrase.md [hunk 1]
```diff
@@ -33,12 +33,12 @@ If you [configured your SSH passphrase with the macOS keychain](/articles/workin
...
+If you lose your SSH key passphrase, there's no way to recover it. You'll need to [generate a brand new SSH keypair](/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) or [switch to HTTPS cloning](/github/getting-started-with-github/managing-remote-repositories) so you can use a Personal Access Token instead.
 
 {% endwindows %}
 
 {% linux %}
 
-If you lose your SSH key passphrase, there's no way to recover it. You'll need to [generate a brand new SSH keypair](/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) or [switch to HTTPS cloning](/github/getting-started-with-github/about-remote-repositories/#cloning-with-https-urls) so you can use your GitHub password instead.
+If you lose your SSH key passphrase, there's no way to recover it. You'll need to [generate a brand new SSH keypair](/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) or [switch to HTTPS cloning](/github/getting-started-with-github/about-remote-repositories/#cloning-with-https-urls) so you can use a Personal Access Token instead.
```
Comment (isaacmbrown) [inline content/authentication/troubleshooting-ssh/recovering-your-ssh-key-passphrase.md:42]:
```suggestion
If you lose your SSH key passphrase, there's no way to recover it. You'll need to [generate a brand new SSH keypair](/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) or [switch to HTTPS cloning](/github/getting-started-with-github/about-remote-repositories/#cloning-with-https-urls) so you can use a {% data variables.product.pat_generic %} instead.
```

**Comment by isaacmbrown** [comment]
Hi @lxjv, thanks for opening this PR ✨ You're right, this should say "personal access token"

We now use a variable for the phrase "personal access token," so I've updated your PR to use this variable and also fixed a link.

I'll check this looks good on staging and approve this to be merged.

**Comment by lxjv** [comment]
alright! thanks for fixing it :)

**Review by isaacmbrown — APPROVED** [review]
Looks good on staging!

**Pull request merged**